src/GovUk.Education.ExploreEducationStatistics.Admin/Services/FootnoteService.cs

```diff
@@ -70,21 +55,20 @@ public class FootnoteService : IFootnoteService
                 .OnSuccess(_userService.CheckCanUpdateRelease)
                 .OnSuccess(async () =>
                 {
-                    var footnote = (await _context.Footnote.AddAsync(new Footnote
+                    var newFootnoteId = Guid.NewGuid();
```

Comment:
This should be a sequential guid in the same way it would be if you left entity framework to generate it.